freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/scale/ScalingPath.java

```diff
@@ -0,0 +1,58 @@
+package com.sequenceiq.freeipa.api.v1.freeipa.stack.model.scale;
+
+import java.util.Objects;
+
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.common.FormFactor;
+
+public class ScalingPath {
+
+    private FormFactor originalFormFactor;
+
+    private FormFactor targetFormFactor;
```

Comment:
as setters are not used (And I don't think it would make much sense to use them), these should be final